Week-2/Basic CSS/Auto Loan/style.css

```diff
@@ -0,0 +1,62 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap");
```

Comment:
Import this font inside of the HTML document as a link tag instead of the import function in CSS.

Reason being that the import statement blocks the main thread and creates performance bottlenecks in rendering process.